include/beman/inplace_vector/inplace_vector.hpp

```diff
@@ -546,7 +571,12 @@ class inplace_vector : public inplace_vector_base<T, Capacity> {
 
   template <class... Args>
   constexpr reference unchecked_emplace_back(Args &&...args) {
+#ifdef __cpp_constexpr_dynamic_alloc
     auto final = std::construct_at(end(), std::forward<Args>(args)...);
+#else
+    // Note placement new may not be not constexpr friendly
```

Comment:
```suggestion
    // Note: placement new may not be non-constexpr friendly
```

I think this is what was intended?

Reply:
Ops typo here, placement new is not constexpr friendly. Thanks for double checking 😅